applications/nrf_desktop/README.rst

```diff
@@ -295,7 +295,7 @@ The report is used by the :ref:`nrf_desktop_config_channel`.
 
 .. note::
    The nRF Desktop also uses a dedicated HID output report to forward the :ref:`nrf_desktop_config_channel` data through the nRF Desktop dongle.
-   This report is handled using the configuration channel's infrastructure and it can be enabled using :kconfig:option:`CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT`.
+   This report is handled using the configuration channel's infrastructure and it can be enabled using :ref:`CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT <config_desktop_app_options>`.
```

Comment:
The change is fine, please just update missing places:

**doc/ble_bond.rst:247:** You can set :kconfig:option:`CONFIG_DESKTOP_BLE_USE_DEFAULT_ID` to make the nRF Desktop peripheral initially use the default Bluetooth local identity for the application local identity with ID ``0``.
**doc/ble_bond.rst:259:** :kconfig:option:`CONFIG_DESKTOP_BLE_DONGLE_PEER_ERASE_BOND_BUTTON` - If you want to enable erasing peers using buttons.
**doc/ble_bond.rst:260:** :kconfig:option:`CONFIG_DESKTOP_BLE_DONGLE_PEER_ERASE_BOND_CONF_CHANNEL` - If you want to enable erasing peers using `Configuration channel options`_.

Reply:
fixed.